<launch>
 	<arg default="/dev/ttyUSB0" name="usbPort"></arg>
    <arg default="6a39" name="targetDeviceId"></arg>
    <arg default="POSITIONING_ALGORITHM_UWB_ONLY" name="algorithm"></arg>
    <arg default="DIMENSION_2_5D" name="dimension"></arg>
    <arg default="1200" name="height"></arg>
    <node name="PozyxPositionReaderPython" output="screen" pkg="gtec_uwb_ranging" type="PozyxPositionReaderPython.py">
        <param name="usbPort" value="$(arg usbPort)"></param>
        <param name="targetDeviceId" value="$(arg targetDeviceId)"></param>
        <param name="algorithm" value="$(arg algorithm)"></param>
        <param name="dimension" value="$(arg dimension)"></param>
        <param name="height" value="$(arg height)"></param>
    </node>
</launch>